packages/forms-web-app/src/controllers/lpa-dashboard/appeal-details.js

```diff
@@ -1,35 +1,48 @@
-const { getAppealByLPACodeAndId } = require('../../lib/appeals-api-wrapper');
+const {
+	getAppealByLPACodeAndId,
+	getAppealDocumentMetaData
+} = require('../../lib/appeals-api-wrapper');
 const {
 	VIEW: {
 		LPA_DASHBOARD: { DASHBOARD, APPEAL_DETAILS }
 	}
 } = require('../../lib/views');
-const {
-	constants: { APPEAL_ID }
-} = require('@pins/business-rules');
+
 const dateFns = require('date-fns');
 
+const HAS_APPEAL_TYPE = 'Householder (HAS) Appeal';
+
 const calculateQuestionnaireDueDate = (appeal) => {
 	const { appealValidDate } = appeal;
 	let questionnaireDueDate = {};
-	switch (appeal.type) {
-		case APPEAL_ID.HOUSEHOLDER:
-			questionnaireDueDate = dateFns.addBusinessDays(appealValidDate, 5);
+	switch (appeal.appealType) {
+		case HAS_APPEAL_TYPE:
+			questionnaireDueDate = dateFns.addBusinessDays(dateFns.parseISO(appealValidDate), 5);
```

Comment:
we should probably aim to be consistent with how we are handling questionnaire due date - this is different from implementation on the your-appeals page.

I might have missed something, but would be more straightforward to use `questionnaireDueDate` in appeal case data?

Reply:
This was per a discussion with Abby where she informs me the due date varies by appeal type, so I agreed to code defensively.

Reply:
I meant more how it is being calculated - you could just use the questionnaireDueDate property of the appeal case data returned from the api. Don't worry about it now - can sort in future tickets.